latency/Cargo.toml

```diff
@@ -38,7 +38,7 @@ env_logger = "0.9.0"
 log = "0.4.14"
 rand = "0.8.3"
 slab = "0.4.2"
-clap = { version = "3.1.1", features = ["derive"] }
+clap = { version = "3.1.5", features = ["derive"] }
```

Comment:
This small change is actually a tiny uncompleted part of the previous commit. Typically it won't be an independent commit. Suggest to amend the change in the previous commit to make the log cleaner.

Reply:
Ok, commits squashed.